### ISSUE TITLE
Impaired spatial orientation

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6804]
Preferred term label:
Impaired spatial orientation

Definition (free text, please give PubMed ID)
PMID: 32986636

Parent term (use hpo.jax.org/app)
No found because of the specificity of the disease

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
ORPHA:210272

**Comment by pnrobinson** [comment]
In the article, the authors are describing dizziness, which corresponds to 
https://hpo.jax.org/app/browse/term/HP:0002321

Individuals with MdDS may concurrently experience dizziness, defined as, “a sensation of disturbed or impaired spatial orientation without a false or distorted sense of motion.” (PMID: 32986636)

I do not think this justifies a new term, but if this is incorrect, please reopen with a definition that defines the difference to Vertigo. Thanks!
